### ISSUE TITLE
Docs look bad on mobile

### ISSUE THREAD

**Issue by KronosTheLate** [JuliaAI/StatisticalMeasures.jl#13]
I believe this is a problem with the overal JuliaML webpage theme, but the documentation landing page has a bad look on mobile:
![Screenshot_20231005-181530](https://github.com/JuliaAI/StatisticalMeasures.jl/assets/61620837/5f977f1f-3ef8-4fab-88bb-01ef4ff03a1e)

The collors look kind of retro, and with the bad page-breaks, I find the look very... the opposite of modern.

**Comment by KronosTheLate** [comment]
Absolutely. Is there common organiztion, since they share landing page? Or is it more haphazard than that? It would be nice if there was e.g. one repo that defined a style that the others would use, for easier upgrading while staying consistent.

But you say this is a documenter isse? Should I perhaps open up an issue there? I generally find the vanilla documenter pages work fine on mobile, so it seems specific to the style used here.

**Comment by ablaom** [comment]
> But you say this is a documenter issue?

No, not a Documenter issue. It's just the index.md pages has some raw html code to style the headings. The other pages should just display fine on a mobile. 

If someone wants to instigate some standardisation, I'm happy to support it. It's not a personal priority.
